### PR TITLE
Re-write of the MDCB docs

### PR DESCRIPTION
**Pull request by andyo-tyk**
New content for the first page of the MDCB documentation - including links to nowhere in "Next Steps" (pages that need to be created)

### CODE REVIEW

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- Tyk’s management platform used to control the creation of API configurations, policies and keys in a persistent manner. It provides analytic information on the traffic the Gateways have processed which includes aggregated API usage and detailed information per transaction
+
+**Tyk Pump**
+- Tyk’s open source analytics purger that can be used to export transaction logs from the Tyk deployment to the visualisation tool or other data store of your choice
+
+**Developer Portal**
+- Access point for your API Consumers where you publish your API catalogue(s) and they obtain API keys
+
+
+## Tyk Controller
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: I don't think we use the term "controller" outside tyk cloud.
We usually use the term "control place data centre" or just "control plane"

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+## Overview
+
+Here we will give an overview of the main elements of an MDCB solution, clarifying the terminology used by Tyk.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: we don't explain "mdcb" but use it already. As such, maybe we should move the mdcb bullet to the top of this list?

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+- the workhorse of any deployment, Tyk’s lightweight Open Source API gateway that exposes your APIs for consumption by your users. It is a reverse proxy that secures your APIs, manages session and policies, monitors, caches and manipulates requests/responses when needed before/after it proxies them to and from the upstream.
+
+**Redis**
+- an in-memory data store used as a database, cache and message broker. We use it as pub/sub broker for inter-Gateway communication, and as a cache for API configurations, keys, certificates, and temporary store for analytics records.
+
+**MongoDB/SQL**
+- a persistent data store for API configurations, policies, analytics and aggregated analytics, Dashboard organisations, configurations, dashboard users, portal developers and configuration
+
+**Multi Data Centre Bridge (MDCB)**
+- the backbone of the distributed Tyk deployment, connecting the different Gateway clusters back to the management cluster
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
- the backbone of the distributed Tyk deployment, connecting the different Gateway clusters back to the management control plane.
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md [hunk 4]
```diff
@@ -11,116 +11,118 @@ aliases:
     - /getting-started/tyk-components/mdcb/
 ---
 
-## Introduction
-Tyk Multi Data Centre Bridge (MDCB) acts as a broker between Tyk Gateway Instances that are isolated from one another and typically have their own Redis DB.
+Tyk’s Multi Data Centre Bridge (MDCB) is a separately licensed extension to the Tyk control plane that performs management and synchronisation of logically or geographically distributed clusters of Tyk API Gateways.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md]: Don't you want to use titles like "## Introduction"? this text is at the top of the page and looks like it is more readable with a title.

**Comment by letzya** [comment]
@andyo-tyk can you please look at the diff between the existing version and yours and confirm you want to remove all this content and links? 
Also - it's best if we have at least 3 links in a page to connect the users to other options

**Review by letzya — CHANGES_REQUESTED** [review]
Per my comments throughout Thr text

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md (hunk 4 above)
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md]: `is a separately licensed extension to the Tyk control plane` - repeated again in line 50

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md [hunk 5]
```diff
@@ -11,116 +11,118 @@ aliases:
...
-MDCB is a separately licensed product that allows you to set up Tyk in the above manner.
+When your users are spread geographically and want to access your APIs from different parts of the world you can optimise the performance, value and utility of your APIs by deploying API Gateways in data centres local to them.
 
-### How Tyk MDCB Works
+<img width="1920" alt="MDCB-Globe-Diagram1" src="https://user-images.githubusercontent.com/99968932/204763637-968d44cb-3479-4a5c-95e3-7ac0799e59e1.png">
 
-Tyk MDCB creates a bridge between a configuration source (MongoDB and a centralised Redis DB) and multiple Tyk Gateway instances, this bridge provides API Definitions, Policy definitions and Organization (Tenant) rate limits, as well as acting as a data sink for all analytics data gathered by worker Tyk Gateways.
+Having localised gateways offers benefits to you and your users, such as:
 
-The communication between instances works through a compressed RPC TCP tunnel between the gateway and MDCB, it is incredibly fast and can handle 10,000's of transactions per second.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md]: @andyo-tyk can we please make sure this info goes somewhere?

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+## Overview
+
+Here we will give an overview of the main elements of an MDCB solution, clarifying the terminology used by Tyk.
+
+<image - Tyk MDCB highlevel (controller + workers)>
+
+**Tyk Gateway**
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
### Tyk Gateway
```

Comment (letzya) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: I suggest using header definitions instead of bold so we have a formated structure. Same for the following titles

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
+## Overview
+
+Here we will give an overview of the main elements of an MDCB solution, clarifying the terminology used by Tyk.
+
+<image - Tyk MDCB highlevel (controller + workers)>
+
+**Tyk Gateway** 
+- the workhorse of any deployment, Tyk’s lightweight Open Source API gateway that exposes your APIs for consumption by your users. It is a reverse proxy that secures your APIs, manages session and policies, monitors, caches and manipulates requests/responses when needed before/after it proxies them to and from the upstream.
+
+**Redis**
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
### Redis
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+- an in-memory data store used as a database, cache and message broker. We use it as pub/sub broker for inter-Gateway communication, and as a cache for API configurations, keys, certificates, and temporary store for analytics records.
+
+**MongoDB/SQL**
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
### MongoDB/SQL
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+- a persistent data store for API configurations, policies, analytics and aggregated analytics, Dashboard organisations, configurations, dashboard users, portal developers and configuration
+
+**Multi Data Centre Bridge (MDCB)**
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
### Multi Data Centre Bridge (MDCB)
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+- the backbone of the distributed Tyk deployment, connecting the different Gateway clusters back to the management cluster
+
+**Tyk Dashboard**
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
### Tyk Dashboard
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+- Tyk’s management platform used to control the creation of API configurations, policies and keys in a persistent manner. It provides analytic information on the traffic the Gateways have processed which includes aggregated API usage and detailed information per transaction
+
+**Tyk Pump**
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
### Tyk Pump
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+- Tyk’s open source analytics purger that can be used to export transaction logs from the Tyk deployment to the visualisation tool or other data store of your choice
+
+**Developer Portal**
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
### Developer Portal
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 1 above)
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
## Tyk Control plane
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 13]
```diff
@@ -0,0 +1,59 @@
...
+- Access point for your API Consumers where you publish your API catalogue(s) and they obtain API keys
+
+
+## Tyk Controller
+
+The Controller cluster is where the components required to create MDCB’s shared control plane are located and must consist of the following elements:
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
The control plane contains consists of the following elements:
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+- Tyk Dashboard (used to configure and control the whole Tyk installation)
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
- **Tyk Dashboard** (used to configure and control the whole Tyk installation)
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+- Tyk Gateway (this is refered to as the Management/Controller GW and will not service API transactions; it is important to ensure this is not public facing)
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
- **Tyk Gateway** - This gateway is configured without tags/segments so it can load all the available APIs. For this reason, it is referred to as `a management gateway`. It is used for keys and certificates creations and does not service API requests. As such, it is important to ensure there's no public access to it.
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+- Tyk MDCB
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
- **Tyk MDCB**
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 17]
```diff
@@ -0,0 +1,59 @@
...
+- High availability Redis data store (used as primary reference for the deployment)
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
- **Redis** - High availability Redis data store. This deployment should be backed up in case of failure. (Check this [doc]{{< ref "https://redis.io/docs/management/persistence/) for recommendation on Redis persistency >}}
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 18]
```diff
@@ -0,0 +1,59 @@
...
+- A high availability persistent data store (e.g. MongoDB or SQL replica set)
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
- **MongoDB or SQL** - A persistent data store. Should be deployed and set up for redundancy and high availability 
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 19]
```diff
@@ -0,0 +1,59 @@
...
+To improve resilience and availability, multiple instances of each Tyk component should be deployed and load balanced within the cluster.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
<br>
To improve resilience and availability, multiple instances of each Tyk component should be deployed and load-balanced within the cluster.
```

Comment (letzya) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: Without this break this line continues the previous bullet https://github.com/TykTechnologies/tyk-docs/pull/2093/files?short_path=c4271bf#diff-c4271bf7c2fc706dc8a930992ce43b562dd17b714bbab72251e4c171f78b83a8

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 20]
```diff
@@ -0,0 +1,59 @@
...
+
+<image - controller cluster>
+  
+### Optional Components
+- One or more Tyk Pumps can be deployed within the controller cluster to export transaction logs to your choice of external data sinks for further analytics and visualisation.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
- **Tyk Pump** - Tyk Pumps can be deployed in the control plane to export analytics data (requests/responses) to your [data sink of choice]({{< ref "https://tyk.io/docs/tyk-stack/tyk-pump/other-data-stores/" >}}) for further analytics and visualisation.
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 21]
```diff
@@ -0,0 +1,59 @@
...
+- A Tyk Developer Portal should be deployed within the controller cluster to enhance the end user experience when accessing your APIs; multiple Developer Portals can be deployed - for example for internal and external consumers.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
- **Tyk Developer Portal** - can be added to enhance the end-user experience when accessing your APIs; multiple Tyk Developer Portals can be deployed, for example for internal and external consumers.
```

Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: `multiple Tyk Developer Portals can be deployed, for example for internal and external consumers.` - @andyo-tyk are you sure about this?

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 22]
```diff
@@ -0,0 +1,59 @@
...
+ 
+## Tyk Worker
+Each Worker Cluster must consist of the following elements:
+- One or more Tyk Gateway instance(s) specifically configured as Workers
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
- **Tyk gateway** - Tyk Gateway that is specifically configured as Workers. To provide resilience and availability deploy multiple Tyk Gateways with a load balancer.
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 23]
```diff
@@ -0,0 +1,59 @@
...
+- A single Redis data store shared by all Gateways in the cluster
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
- **Redis** - A single Redis data store shared by all Gateways in the cluster. If you want this data centre to be resilient, available, and independent from its control plane during a disconnection event, it is advised to keep it persistent.
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 24]
```diff
@@ -0,0 +1,59 @@
...
+To improve resilience and availability, multiple instances of each Tyk component should be deployed and load balanced within the cluster.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: not really needed since it was already mentioned above

Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 25]
```diff
@@ -0,0 +1,59 @@
...
+
+<image - worker cluster>
+  
+### Optional Components
+- One or more Tyk Pumps can be deployed within the worker cluster to export transaction logs to your choice of external data sinks for further analytics and visualisation.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion

- **Tyk Pump** - Optional. Tyk Pump that is specifically configured as [hybrid pump]({{< ref "release-notes/version-2.8/#custom-analytics-storage-engines-for-multi-cloud--enterprise-mdcb-users" >}}) can be deployed in the worker data centre. This pump will send analytics data (requests/responses) to external data sinks from within this local/regional data center and will help with further analytics and visualisation. 
```

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 2 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: I don't think we should move the MDCB bullet to the top. I went straight to the abbreviation as this is the second page where we go into more detail looking inside the clusters that were introduced on the first page... however happy to add the full text first.

**Comment by andyo-tyk** [comment]
> Also - it's best if we have at least 3 links in a page to connect the users to other options

I've got links at the bottom of each page - I was hoping that Mark would sort out directing them to the right pages as I wasn't sure what URLs to use.

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 6 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: OK

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 3 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: I'm trying to be consistent with terminology - and specifically I am refering to clusters of components that are distributed (potentially in different data centres/regions) that carry API traffic and the fact that they all connect back to a single cluster of components that does the management.
So I don't disagree with you that the function of the "management cluster" is to provide the "control plane", but here I'm deliberately choosing to use the word cluster.
I've changed "gateway cluster" to "worker cluster" for consistency with the following sections.
I hope this makes sense.

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 1 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: I've modified this to Management Cluster to match the usage further up the page. Just trying to be consistent on this page. It can be changed easily later as and when a decision is made on naming. My preference is that the Tyk Gateway is just called Tyk Gateway - and it is the cluster/deployment that is assigned Controller/Worker or Management/Worker or Control Plane/Data Plane or whatever naming we come up with....

Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: Changed to "Management Cluster"

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 13 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: I agree, this was way too wordy. I've made some changes in common with the terminology used on the page.

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 14 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: I'm not sure it needs to be in bold, but have added this as requested.

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 15 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: I thought to keep this simple and have removed all mention of it being "refered to as the management gateway". I have merged in your wording about keys/certificates.

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 17 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: The suggested format for the link doesn't look right in the preview, so I've just gone for `[doc]("link")` .

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 20 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: As above, I've simplified the format of the link - and used the name Management Cluster for consistency.

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 24 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: I'd prefer to keep the notes about resilience/availability in a separate note below the list of components (for consistency with the Management Cluster above). I've added your comments about the Redis config.

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 21 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: That's my understanding, yes. But I'll remove it given your uncertainty.

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 22 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: I'd prefer to keep the note about resilience/availability in a separate note below the list of components (for consistency with the Management Cluster above).

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 23 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: I'd prefer to keep the note about resilience/availability in a separate note below the list of components (for consistency with the Management Cluster above).

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 25 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: I've embedded the full URL to the release notes, hope that's the right thing to do.

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md (hunk 4 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md]: Added

Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md]: Good spot - it was originally only on line 50 but then I added the introduction sentence later and forgot to remove it. I've merged line 50 into this one.

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md (hunk 5 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md]: I plan to have a separate section for "Advanced MDCB" - this page is intended as an introduction to MDCB following the concept of progressive disclosure.

**Review by andyo-tyk — COMMENTED** [review]
I've added comments to your suggestions.

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md (hunk 5 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md]: I've added this in the new page:
tyk-docs/content/tyk-stack/tyk-multi-data-centre/advanced-mdcb/mdcb-technical-overview.md
(covered by draft PR #2145)

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md (hunk 17 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]: I've now seen the Confluence advice on documentation and understand the <<ref>> markdown. I will revise the links.

**Comment by andyo-tyk** [comment]
Changes made to correct the link style (to use ({{< ref ““ >}}) format).

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md [hunk 26]
```diff
@@ -0,0 +1,60 @@
...
+### MongoDB/SQL
+- A persistent data store for API configurations, policies, analytics and aggregated analytics, Dashboard organisations, configurations, dashboard users, portal developers and configuration.
+
+
+## Management Cluster
+<img width="1500" alt="Management_cluster" src="https://user-images.githubusercontent.com/99968932/211528518-c52f99ca-5794-48e1-8e7f-9d2e76edb27f.png">
+
+The Management Cluster must consist of the following elements:
+- **Tyk Dashboard** (used to configure and control the whole Tyk installation)
+- **Tyk Gateway** (used for creation of keys and certificates, this does not service API requests; it is important to ensure there is no public access to it and it must not be tagged, zoned or sharded as it "belongs" to the whole Tyk installation)
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-components.md]:
```suggestion
- **Tyk Gateway** (used for creation of keys and certificates, this does not service API requests; it is important to ensure there is no public access to it and it must not be shared (tagged) as it "belongs" to the whole Tyk installation)
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md [hunk 27]
```diff
@@ -12,115 +12,117 @@ aliases:
...
 
-### Organisational Benefits
+- You can control geographic distribution of traffic, restricting traffic to data centres/regions of your choice.
+- You can put your Tyk API Gateways close to users, but still have a single management layer.
+- You have a single, simple, point of access for configuration of your complex API infrastructure and yet deploy multiple Developer Portals, if required, to provide access to different user groups (e.g. Internal and External).
+- You can physically segment teams and environments within a single data centre, giving each team full control of its own cluster and server resources without the noisy neighbours you might experience in a standard self-managed deployment.
+- You can deploy gateways with whichever mix of cloud vendors you wish.
+- You can mix and match cloud and on premises data centres.
 
-MDCB-worker gateways are tied to a single organisation in the Dashboard. This means that you can set up different teams as organisations in the Dashboard, and each team can run it's own set of Gateways that are logically isolated.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md]: This content is super important. Do we cover it somewhere?

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md [hunk 28]
```diff
@@ -12,115 +12,117 @@ aliases:
...
-Also, the lookup to MDCB is via a resilient compressed RPC channel that is designed to handle ongoing and unreliable connectivity, it is also encrypted, and so safer to use over the open internet or inter-DC links.
+### Flexible architecture
 
-### Organisational Benefits
+- You can control geographic distribution of traffic, restricting traffic to data centres/regions of your choice.
+- You can put your Tyk API Gateways close to users, but still have a single management layer.
+- You have a single, simple, point of access for configuration of your complex API infrastructure and yet deploy multiple Developer Portals, if required, to provide access to different user groups (e.g. Internal and External).
+- You can physically segment teams and environments within a single data centre, giving each team full control of its own cluster and server resources without the noisy neighbours you might experience in a standard self-managed deployment.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md]:
```suggestion
- You can physically [segment teams and environments]({{< ref "/advanced-configuration/manage-multiple-environments/with-tyk-multi-cloud.md" >}}) within a single data centre, giving each team full control of its own cluster and server resources without the noisy neighbours you might experience in a standard self-managed deployment.
```

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md (hunk 27 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/tyk-multi-data-centre.md]: I think this should go in the proposed new "Advanced MDCB" section where we can explain in more detail this use case and how to configure it. I find the original wording unclear.

**Comment by letzya** [comment]
> Changes made to correct the link style (to use ({{< ref ““ >}}) format).

Cannot see it in the diff

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-example-minimising-latency.md [hunk 29]
```diff
@@ -0,0 +1,134 @@
...
+1. `./down.sh -n tyk-worker-us`
+2. `/down.sh -n tyk-worker-eu`
+3. `/down.sh`
+
+Don’t forget to tear down your clusters in GCP if you no longer need them!
+
+---
+
+Next Steps
+ - [Advanced MDCB]({{< ref "/tyk-stack/tyk-multi-data-centre/advanced-mdcb/advanced-mdcb.md" >}})
```
Comment (sredxny) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-example-minimising-latency.md]: @andyo-tyk this file doesnt seems to exist. Where did you wanted to redirect the user?

Comment (andyo-tyk) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/mdcb-example-minimising-latency.md]: That's a new file - adding in a separate PR (mdcb-docs-rewrite-pt2). I'll delete that line and add it back in in the second PR.

File: tyk-docs/content/mdcb-example-minimising-latency.md [hunk 30]
```diff
@@ -0,0 +1,134 @@
...
+---
+
+## Overview
+As described previously, Acme Global Bank has operations and customers in both the EU and USA.
+
+To decrease the latency in response from their systems and to ensure that data remains in the same legal jurisdiction as the customers (data residency), they have deployed backend (or, from the perspective of the API gateway, “upstream”) services in two data centres: one in the US, the other in the EU.
+
+Without MDCB, Acme Global Bank would deploy a Tyk Gateway cluster in each data centre and then have the operational inconvenience of maintaining two separate instances of Tyk Dashboard to configure, secure and publish the APIs.
+
+By deploying an MDCB Controller, however, they are able to centralise the management of their API Gateways and gain resilience against failure of different elements of the deployment - worker gateways or control plane - improving availability of their public APIs.
```
Comment (letzya) [inline tyk-docs/content/mdcb-example-minimising-latency.md]:
```suggestion
By using Tyk's Multi-Data Centre Bridge (MDCB), however, they are able to centralise the management of their API Gateways and gain resilience against failure of different elements of the deployment - worker gateways or control plane - improving the availability of their public APIs.
```

File: tyk-docs/content/mdcb-example-minimising-latency.md [hunk 31]
```diff
@@ -0,0 +1,134 @@
...
+tags: ["MDCB", "latency"]
+description: "How to minimize latency in worker datacenters."
+---
+
+## Overview
+As described previously, Acme Global Bank has operations and customers in both the EU and USA.
+
+To decrease the latency in response from their systems and to ensure that data remains in the same legal jurisdiction as the customers (data residency), they have deployed backend (or, from the perspective of the API gateway, “upstream”) services in two data centres: one in the US, the other in the EU.
+
+Without MDCB, Acme Global Bank would deploy a Tyk Gateway cluster in each data centre and then have the operational inconvenience of maintaining two separate instances of Tyk Dashboard to configure, secure and publish the APIs.
```
Comment (letzya) [inline tyk-docs/content/mdcb-example-minimising-latency.md]:
```suggestion
Without a dedicated solution for this multi-regions use case, Acme Global Bank would deploy a Tyk Gateway cluster in each data centre and then have the operational inconvenience of maintaining two separate instances of Tyk Dashboard to configure, secure and publish the APIs.
```

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/content/mdcb-example-minimising-latency.md (hunk 30 above)
Comment (andyo-tyk) [reply tyk-docs/content/mdcb-example-minimising-latency.md]: Makes sense.

File: tyk-docs/content/mdcb-example-minimising-latency.md (hunk 31 above)
Comment (andyo-tyk) [reply tyk-docs/content/mdcb-example-minimising-latency.md]: Makes sense.

**Review by letzya — APPROVED** [review]
